core/geotime/src/test/java/org/locationtech/geowave/core/geotime/adapter/SpatialFieldMapperTest.java

```diff
@@ -0,0 +1,40 @@
+package org.locationtech.geowave.core.geotime.adapter;
+
+import static org.junit.Assert.assertEquals;
+import org.geotools.referencing.CRS;
+import org.junit.Test;
+import org.locationtech.geowave.core.geotime.store.dimension.SpatialField.SpatialIndexFieldOptions;
+import org.locationtech.geowave.core.index.persist.PersistenceUtils;
+import org.locationtech.geowave.core.store.adapter.FieldDescriptor;
+import org.locationtech.jts.geom.Geometry;
+import org.opengis.referencing.FactoryException;
+import com.google.common.collect.Lists;
+
+public class SpatialFieldMapperTest {
+
+  @Test
+  public void testGeometryFieldMapper() throws FactoryException {
+    FieldDescriptor<Geometry> testField =
```

Comment:
it'd be good to test lat and lon as numbers too here

Reply:
Done